src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -392,7 +392,7 @@ cdef class Face:
     def upem(self, value: int):
         hb_face_set_upem(self._hb_face, value)
 
-    def reference_blob(self) -> Blob:
+    def get_blob(self) -> Blob:
```

Comment:
I’d even call it `blob` and make it a property.

Reply:
Sounds good.